CascadingSelectsDemo/Controllers/GameSystemController.cs

```diff
@@ -14,34 +11,10 @@ public ActionResult Games(string developer, string console)
             string[] games = new[] { "" };
```

Comment:
Being that GamesHelper.SelectGames() already guards against invalid data, you could probably just:

``` csharp
return Json(GamesHelper.SelectGames(developer, console), JsonRequestBehavior.AllowGet);
```